pkgs/by-name/ma/marksman/package.nix

```
@@ -3,23 +3,22 @@
   fetchFromGitHub,
   buildDotnetModule,
   dotnetCorePackages,
-  marksman,
-  testers,
+  versionCheckHook,
 }:
 
-buildDotnetModule rec {
+buildDotnetModule (finalAttrs: rec {
```

```suggestion
buildDotnetModule (finalAttrs: {
```

I believe the `rec` isn't suppose to be there with `finalAttrs`.